abnf/odata-aggregation-abnf.txt

```diff
@@ -132,13 +132,17 @@ pathSegment     = ( complexProperty / complexColProperty ) [ "/" qualifiedComple
 
 groupingProperty = grpPropPathPrefix
                    ( entityNavigationProperty [ "/" qualifiedEntityTypeName ]
-                   / primitiveProperty 
+                   / entityAnnotationInQuery  [ "/" qualifiedEntityTypeName ]
+                   / primitiveProperty        [ "/" primitiveAnnotationInQuery ]

```

Comment:
I would have expected
```suggestion
                   / primitiveProperty
                   / primitiveAnnotationInQuery
```

Reply:
That would rule out
https://github.com/oasis-tcs/odata-abnf/blob/b464b21ff13ec269cb0f83163301e77b4b1d230b/abnf/odata-aggregation-testcases.yaml#L357

It's more complicated...